vidarrtest-regression.json.in

```diff
@@ -6,7 +6,7 @@
             "variantMerging.combineVariants.priority": "strelka,mutect2",
             "variantMerging.combineVariants.referenceFasta": "$HG19_ROOT/hg19_random.fa",
             "variantMerging.combineVariants.timeout": null,
-            "variantMerging.combineVariants.modules": "hg19/p13 java/9 discvrseq/1.3.21",
+            "variantMerging.combineVariants.modules": "hg19/p13 discvrseq/1.3.21 java/9",
```

Comment:
May benefit moving the reference genome related modules to the workflow, instead of having to define them in the olive?